src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.CSharp.DesignTime.targets

```diff
@@ -22,25 +22,27 @@
     <DefineCSharpItemSchemas>false</DefineCSharpItemSchemas>
   </PropertyGroup>
 
-    <ItemGroup>
-     <PropertyPageSchema Include="$(ManagedXamlResourcesDirectory)CSharp.ProjectItemsSchema.xaml;"/>
+  <ItemGroup>
+    <PropertyPageSchema Include="$(ManagedXamlResourcesDirectory)CSharp.ProjectItemsSchema.xaml;"/>
 
-     <ProjectCapability Include="CSharp;Managed"/>
-    </ItemGroup>
+    <ProjectCapability Include="
+                          CSharp;
```

Comment:
Can you make sure that CPS removes newline when it calculates capabilities?

Reply:
We already do this in our other DesignTime targets.